Tests/ApolloCodegenTests/ApolloCodegenConfigurationCodableTests.swift

```diff
@@ -282,6 +287,22 @@ class ApolloCodegenConfigurationCodableTests: XCTestCase {
       .to(throwError())
   }
 
+  func test__encodeMinimalConfigurationStruct__canBeDecoded() throws {
```

Comment:
Should we have a test for JSON configurations still using the old properties that they continue to decode correctly?

Reply:
I don't think it's that necessary to add additional tests for deprecated things. You're not _wrong_, I just think it's not worth the time. If you want me to do it, I will though haha.

Reply:
It's not required for my approval.